test/extensions/clusters/common/logical_host_integration_test.cc

```diff
@@ -60,7 +62,7 @@ TEST_F(LogicalHostIntegrationTest, LogicalDNSRaceCrashTest) {
                              "::8",
                              "::9",
                          }));
-            address = (address + 1) % 128;
+            *address_ptr = (*address_ptr + 1) % 128;
```

Comment:
nit: s/(*address_ptr + 1)/(address + 1)/ since you already have that in a temp.

Reply:
done